app/models/reports/repository.rb

```diff
@@ -155,17 +163,46 @@ def initialize(regions, periods:)
     end
 
     memoize def controlled
```

Comment:
So if I understand this correctly, we only compare the two on requests for v1? I assume this is because it's the more common code path and will provide the broadest coverage. 

But if that's the case, should we be raising on mismatches? Seems like that would potentially cause errors to users who aren't using the new v2 flag. I looked at `raise_on_mismatches` in the scientist docs, and it says to only use it in your test suite. So I'm unsure if I'm understanding this correctly.

Reply:
I looked into using Diffy briefly, and it didn't really give us anything useful out of the box...I'll leave the diffing for a future PR, as there are lots of options out there.

Reply:
Cool, yeah, diffing in another PR sounds good. We'll also know by then if it's necessary.